NOMIS/nomis.go

```diff
@@ -301,3 +307,25 @@ func downloadFile() {
 
 	fmt.Printf("Downloaded a file %s with size %d", fileName, size)
 }
+
+func checkSubString (res CenStructure, index0, indx int)  string {
+	var strArray []string
+	var result string
+	value :=strings.Contains(res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[indx].Text.(string), "http")
```

Comment:
I'd be inclined to set a variable using `res.Structure.Keyfamilies.Keyfamily[index0].Annotations.Annotation[indx].Text.(string)` - that way its only referenced once in the function.